src/books/dto/books.create.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class BooksCreateDto {
+  name: string;
```

Comment:
Mấy field của dto thì thêm cái này vào, có thể sẽ hơi khác với enum, array, optional...

```suggestion
  @ApiProperty()
  name: string;
```